_posts/elements/2015-04-15-icons.md

```diff
@@ -0,0 +1,738 @@
+---
+layout:       element
+title:        Icons
+date:         2015-04-15 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- icons
+
+slug:         icons
+scssurl:      ""
+
+description:  ...
+
+info: ...
+---
+
+<h3 class="hd-6">Media</h3>
+<div class="example-set">
+    <div class="swatch">
```

Comment:
NIT: this isn't really a "swatch". That label was meant for colors. Can you sync up class names here so that there's a more universal class for any examples and perhaps one for specific uses (like color swatches). Here's what I'm thinking:

```html
<!-- basic example -->
<div class="example"> ... </div>

<!-- color example -->
<div class="example example-color"> ... </div>

<!-- icon example -->
<div class="example example-icon"> ... </div>
```